.github/scripts/build.sh

```diff
@@ -20,6 +20,10 @@ if [ "$RUN_BOOTSTRAP" = 1 ]; then
     .github/scripts/install-dependencies.sh
 fi
 
+if [ "$UPDATE_SOURCE" = 1 ]; then
+    .github/scripts/update_source.sh
```

Comment:
I'd use plural (`update_sources.sh`).

Reply:
also plural for UPDATE_SOURCE?

Reply:
It would make sense to have both cosistent.